app/controllers/application_controller.rb

```diff
@@ -255,14 +255,14 @@ def form_permissions?(form:)
     return false if form.blank?
     return true if admin_permissions?
     return true if current_user.has_role?(:form_manager, form)
+    return true if form_approver_permissions?
```

Comment:
this elevates the Organizational Form Approver to Form Manager status, as opposed to Response Viewer status.
this enables them to see all Form tabs